lib/MqttClient.js

```diff
@@ -419,6 +422,40 @@ MqttClient.prototype.handleCustomCommand = function (message) {
                     console.info("Missing spot_id");
                 }
                 break;
+
+            /**
+             * {
+             *   "command": "go_to",
+             *   "spot_id": "Somewhere"
+             * }
```

Comment:
Comment should be edited:
/**
             * {
             *   "command": "store_map",
             *   "name": "Somemapname"
             * }
             */